tooling/autodispose-rx-error-prone-checker/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+  id "net.ltgt.errorprone" version "0.0.13"
+  id "com.github.johnrengelman.shadow" version "2.0.1"
+  id "com.github.johnrengelman.plugin-shadow" version "2.0.0"
+  id "java"
+}
+
+// we use this config to get the path of the JDK 9 javac jar, to
+// stick it in the bootclasspath when running tests
+configurations.maybeCreate("epJavac")
```

Comment:
Many config options in this file are not required. They are specific to nullaway and do not translate here.

Remove this line

Reply:
This actually might be needed for unit tests, to get EP java in bootclasspath. Can try without first.

Reply:
This is needed for unit tests.